api/v1/mongodbcommunity_types.go

```diff
@@ -620,6 +620,30 @@ func (m MongoDBCommunity) LogsVolumeName() string {
 	return "logs-volume"
 }
 
+func (m MongoDBCommunity) IsTLSEnabled() bool {
+	return m.Spec.Security.TLS.Enabled
+}
+
+func (m MongoDBCommunity) IsTLSOptional() bool {
+	return m.Spec.Security.TLS.Optional
+}
+
+func (m MongoDBCommunity) TLSSecretKeyName() string {
+	return "tls.key"
+}
+
+func (m MongoDBCommunity) TLSSecretCertName() string {
+	return "tls.crt"
+}
+
+func (m MongoDBCommunity) TLSSecretPEMName() string {
+	return "tls.pem"
+}
```

Comment:
All of these are constant but wrapped in a function as enterprise might provide different entries